indigo-plugin/build.sc

```diff
@@ -19,6 +19,7 @@ trait IndigoPluginModule extends CrossScalaModule with PublishModule with Scalaf
     crossScalaVersion match {
       case "2.12" => "2.12.20"
       case "2.13" => "2.13.16"
+      case "3"    => "3.3.6" // Relying on LTS Scala
```

Comment:
LTS seems like a sensible move, and yet:

> build.mill and package.mill files are now written in Scala 3.7.1

https://github.com/com-lihaoyi/mill/blob/main/changelog.adoc#major-breaking-changes

🤷 

I've asked if there's a recommendation:
https://discord.com/channels/632150470000902164/1295786923410984981/1404585711981691012